core/src/main/scala/kafka/tools/GetOffsetShell.scala

```diff
@@ -70,7 +70,7 @@ object GetOffsetShell {
                            .withRequiredArg
                            .describedAs("partition ids")
                            .ofType(classOf[String])
-    val timeOpt = parser.accepts("time", "timestamp of the offsets before that. [Note: No offset is returned, if the timestamp greater than recently committed record timestamp is given.]")
+    val timeOpt = parser.accepts("time", "timestamp of the offsets before that. [Note: -1 indicates offset for the specified time is not found]")
```

Comment:
I think the description is not clear. Basically, we defined `-1`  means `latest` in the `time` option. So, I think we should explicitly mention the `returned offset`. ex:
[Note: The returned offset -1 indicates offset for the specified time is not found]

WDYT?

Reply:
That's a nice catch, -1 may be misunderstood to a time arg, your suggestion makes sense to me.